### ISSUE TITLE
For 'pip' instalation, missing requirements

### ISSUE THREAD

**Issue by filmil** [dsoprea/GDriveFS#130]
The following tacit requirements seem to be missing (at least) in readme.md

- get-pip.py
- python-dev

to be able to do

`sudo pip install gdrivefs`

**Comment by m-bra** [comment]
With `python-dev`, do you mean the Ubuntu (maybe even Debian) package? And where to get `get-pip.py`?
Worked for me on Ubuntu 15.04 only after `sudo apt-get install python-dev` at least.

**Comment by dsoprea** [comment]
Should be fine as long as PIP is installed (python-pip under Ubuntu).

It's out of scope to describe how to install PIP on every platform.

It might not be a bad idea to include a *link* to PIP install steps, since,
though this is a Python tool, it is marketed toward the general public.
Though they may be willing to do administrative tasks, they might often be
unfamiliar with this.

Dustin

**Comment by stanelie** [comment]
I guess it's not that obvious if you are not used to using PIP.
I had to install python-dev myself to be able to install gdrivefs, even if I already had PIP installed.
See https://github.com/dsoprea/GDriveFS/issues/150

**Comment by dsoprea** [comment]
Hi, guys. Sorry for my silence.

- I don't describe how to install PIP, because, I'd also have to describe how to install the Linux and Python development environments (*build-essentially* and *python-dev*, respectively, for Ubuntu). Even if the former is plausibly within scope, the latter definitely isn't. *gevent*, which is a tool that is helping us to be able to transfer information in concurrent streams from Google, requires this in order to build during the install.

- To emphasize the point, *python-dev* is required to build *gevent* (used by GDFS). It's not a direct requirement of PIP.

I've augmented the documentation around the install steps. It should at least make the install more straight-forward for Ubuntu users.
